assets/queries/cloudFormation/amplify_app_access_token_rule/query.rego

```diff
@@ -0,0 +1,89 @@
+package Cx
+
+
+CxPolicy [ result ]  {
+
+  document := input.document[i]
+  resource := document.Resources[key]
+  resource.Type == "AWS::Amplify::App"
+
+  properties := resource.Properties
+  paramName  := properties.AccessToken
+  defaultToken := document.Parameters[paramName].Default
+  count(defaultToken) > 50
+  regex.match(`^[A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*$`,defaultToken) #Access Token is a JWT token from following docs: https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-tokens-with-identity-providers.html#amazon-cognito-user-pools-using-the-access-token
```

Comment:
To reduce the number of columns in this file, place this inline comment on the previous line.